### PR TITLE
Rename rbac

### PR DESCRIPTION
**Pull request by vpsx**
PXP-3071 

### Improvements
Rename rbac to authz/abac depending on context. In code, docs, useryamls.

### CODE REVIEW

File: fence/config-default.yaml [hunk 1]
```diff
@@ -451,7 +451,7 @@ HTTP_PROXY:
 
 # //////////////////////////////////////////////////////////////////////////////////////
 # MICROSERVICE PATHS
-#   - Support `/data` endpoints & `/rbac` endpoints
+#   - Support `/data` endpoints & authz functionality
 # //////////////////////////////////////////////////////////////////////////////////////
```
Comment (Avantol13) [inline fence/config-default.yaml]: in the test config there was another change for `role-based access control`, does that exist in this file?

File: fence/blueprints/data/blueprint.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 )
 from fence.errors import Forbidden, InternalError, UserError
 from fence.utils import is_valid_expiration
-from fence.rbac import check_arborist_auth
```
Comment (Avantol13) [inline fence/blueprints/data/blueprint.py]: this was really the only place where that function got imported?

File: fence/config-default.yaml (hunk 1 above)
Comment (vpsx) [reply fence/config-default.yaml]: Oho! Good catch thanks!

File: fence/blueprints/data/blueprint.py (hunk 2 above)
Comment (vpsx) [reply fence/blueprints/data/blueprint.py]: seems like it 😛

**Pull request merged**